client/main.css

```diff
@@ -2858,3 +2885,54 @@ margin-bottom: 0 !important;
     max-height: 100vh;
   }
 }
+.banner-container {
+  width: 100vw;
+  text-align: center;
+  font-size: 16px;
+  z-index: 19;
+  height: 32px;
+}
+.banner {
+  height: 100%;
+}
+.banner.light {
+  background-color: white;
+  color: #333;
+}
+.banner.light a {
+  color: #333;
+}
+.banner.medium {
+  background-color: #5940de;
+  color: white;
+}
+.banner.medium a {
+  color: white;
+}
+.banner.dark {
+  background-color: #333;
+  color: #5940de;
+}
+.banner.dark a {
+  color: #5940de;
+}
+.banner a:hover {
+  text-decoration: underline;
+}
+.banner p {
+  padding: 5px 0;
+  margin: auto;
+}
+@media screen and (min-width: 430px) and (max-width: 567px) {
+  .banner-container {
+    font-size: 14px;
+  }
+}
+@media screen and (max-width: 429px) {
+  .banner-container {
+    font-size: 11.5px;
+  }
```

Comment:
I can't accept this and line 2928. Nicki painstakingly created a type system, and using whatever font sizes outside of it breaks that. See lines 35-40 for acceptable sizes.